src/main/resources/schema/post.graphqls

```diff
@@ -7,6 +7,18 @@ type Post {
     identifier: ID!
     description: String!
     creationDate: String!
+    language: String!
     num_likes: Int!
     num_replies: Int!
+    location_name: String!
```

Comment:
you probably planned to do it later but it should be better to keep the graph nature of the data, eg to render location, owner and creator information into "sub-graphs" of the response.

Reply:
I would like to find a better solution than the one I have in my mind.
I thought to do it programmatically, at the moment when we go through the answer to build the response list to graphql, I think to build a subset for each "sub-graphs" ex:

```java
BindingSet bindingSet = tupleQueryResult.next();
                HashMap<String, String> postResult = new HashMap<>();

                for (Binding binding : bindingSet) {
                    if (binding.getName().contains("-")){
                        String objectName = binding.getName().split("-")[0];
                        String key = binding.getName().split("-")[1];
                        HashMap<String, String> subGraph = new HashMap<>();
                        
                        subGraph.put(key, binding.getValue().stringValue());
                        postResult.put(objectName, binding.getValue().stringValue());
                        ...
                    } else {
                        postResult.put(binding.getName(), binding.getValue().stringValue());
                    }
                }
```
This algorithm is not complete it lacks some details ...